packages/block-library/src/group/edit.js

```diff
@@ -39,10 +48,25 @@ function GroupEdit( { className, setBackgroundColor, backgroundColor } ) {
 				/>
 			</InspectorControls>
 			<div className={ classes } style={ styles }>
-				<InnerBlocks />
+				<InnerBlocks
+					renderAppender={ ! hasInnerBlocks && renderAppender }
```

Comment:
I wouldn't mind to have seen this function be inlined.

Alternatively (separately), I would like to see as proposed in https://github.com/WordPress/gutenberg/pull/14241#discussion_r274571994 with `createElement` being used with `renderAppender` so this could be come:

```
renderAppender={ ! hasInnerBlocks && InnerBlocks.ButtonBlockAppender }
```

Reply:
Thanks for the review. I'll follow up with a PR that tries out `createElement`. The only concern I'd have is lack of consistency with other render props, but that would only be an issue if `renderAppender` ever took arguments.

Reply:
PR is here - https://github.com/WordPress/gutenberg/pull/15259